t/full-stack.t

```diff
@@ -47,6 +47,8 @@ use lib "$FindBin::Bin/lib";
 use Test::More;
 use Test::Mojo;
 use Test::Output 'stderr_like';
+use autodie ':all';
```

Comment:
Is there a reason to activate `autodie` here?

Reply:
yes, to handle yet unhandled errors e.g. when we open files further below?